src/sme_lib/enemy/FireyPetey.cpp

```diff
@@ -304,12 +271,12 @@ bool TNerveFPFireBreath::execute(TSpineBase<TLiveActor> *spine) const
       target->mFire[i]->mVelocity.z = 0.0f;
       target->mFire[i]->mLifetime=0;
     }
-    this_00->setFrameRate((float)SMSGetAnmFrameRate__Fv() * 2.0f,0);
+    peteyMActor->setFrameRate((float)SMSGetAnmFrameRate__Fv() * 2.0f,0);
```

Comment:
Rather than casting a float, just have the function return type be float directly